client/src/components/mapping/FhirMappingPanel/index.tsx

```diff
@@ -48,6 +49,10 @@ const FhirMappingPanel = () => {
     );
   };
 
+  React.useEffect(() =>{
+    dispatch(fetchAvailableExtensions)
```

Comment:
Is this called when a new extension is created?

Reply:
it is now 👍